project_security_group/views/project_security_group_menu.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+<data>
+
+    <!-- Top menu item rewrite-->
+    <menuitem name="Project"
```

Comment:
Don't redeclare entire menu, but use the <record> and <field> tags for overwriting only the desired field.